bbc/simorgh
# Update fontFamilyBase

Update `fontFamilyBase` to be  `Helvetica, Arial, sans-serif`, to match UX requirements. 

* [x] Fixes https://github.com/bbc/simorgh/issues/433
* [x] UX designer approval

**Test steps**
1. `git checkout font-fallbacks`
2. `npm run dev` 
3. Go to https://localhost:7080/article/scenario-27
4. Inspect the Headline
5. See that the font family is `ReithSerifNewsMedium, Helvetica, Arial, sans-serif`
